src/codeu/chat/server/Server.java

```diff
@@ -143,7 +139,30 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
-    // Get Conversations By Id - A client wants to get a subset of the converations from
+    // Get Server Request - A client whats to get all the request to the server from the back end.
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command(){
+        @Override
+        public void onMessage(InputStream in, OutputStream out) throws IOException{
+            Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+
+
+            try{
+              serverInfo = new ServerInfo();
+            }catch (Exception ex){
+              LOG.error(ex, "There was a problem with parsing the ServerInfo.");
+            }
+
+
+            //Writes out the ServerInfo Version and StartTime to the user!
+            Uuid.SERIALIZER.write(out, serverInfo.version);
+            Time.SERIALIZER.write(out, view.getInfo().startTime);
```

Comment:
While I generally understand why you take version from one place and startTime from the view, the code looks weird and 'hard to read'. Consider using just view.getInfo() ... you do not really need to create new serverinfo just for the version if you already have it.